pkg/cmd/roachtest/tests/mixed_version_tenant_span_stats.go

```diff
@@ -88,6 +88,11 @@ func registerTenantSpanStatsMixedVersion(r registry.Registry) {
 				mixedVersionReqError := "unable to service a mixed version request"
 				unknownFieldError := "unknown field"
 
+				// Skip finalizing state.
+				if h.Context().Finalizing {
+					return nil
+				}
+
 				// If we have nodes in mixed versions.
 				if len(h.Context().FromVersionNodes) > 0 && len(h.Context().ToVersionNodes) > 0 {
```

Comment:
I'd also remove this condition here. Now that we don't run this validation when the upgrade is finalizing, we know that the cluster version is still at V22_2 every time this function is called, and therefore we should get the error every time, right?

Reply:
Good point, removed.